### ISSUE TITLE
Challenge http://www.freecodecamp.com/challenges/waypoint-style-text-inputs-as-form-controls has an issue 

### ISSUE THREAD

**Issue by AdventureBear** [FreeCodeCamp/freecodecamp#638]
style the CSS in the instruction paragraph to match other challenges (ie should be red monospaced font, but it's normal paragraph font)

Bonus: Under more indicate what the benefit of this step is (no text in "more" for this challege)

**Comment by terakilobyte** [comment]
Thanks or the suggestion. It'd be great to see a PR... ;)

**Comment by AdventureBear** [comment]
woulldn't it be? ;)   I figured it was better to document what I saw the first time through...and worry about implementation later.  I know from personal experience that if I accept something I feel is not optimal without addressing it somehow, it will either get forgotten or I'll get complacent.   That's the main reason I ran through all the new HTML/CSS sections...bonus..>I learned how to use font awesome!
